src/test/java/com/training/services/tests/ProductsTest.java

```diff
@@ -17,6 +23,9 @@
     Map<String, Object> queryParam;
     String baseURL;
     Response response;
+    final String day="day";
+    final String month="month";
+    final String daysInMonth="daysInMonth ";
```

Comment:
final strings as per java conventions should be all in uppercase like:
DAY
MONTH
DAYS_IN_MONTH

Also make an hobbit of declaring all global variables as private